screwdriver.yaml

```diff
@@ -34,8 +34,12 @@ jobs:
 
   release:
     secrets:
-        - BINTRAY_USER
-        - BINTRAY_API_KEY
+        - GPG_KEYNAME
+        - GPG_PASSPHRASE
+        - GPG_ENCPHRASE
+        - OSSRH_USER
+        - OSSRH_TOKEN
     requires: [~tag, ~release]
     steps:
-      - release: mvn -B -DskipTests deploy --settings ./settings.xml
+      - publish: "screwdriver/scripts/publish.sh"
+      - build: "screwdriver/scripts/build.sh"
```

Comment:
we can remove the build step as we don't do it currently. its probably needed to ensure there are no build/test failures cos publish skips test. But we rely on commit to be successful before manually trigerring release so can be skipped.
If we wanted to add it, it should be before the `publish` step as SD runs the steps in the order they are defined in.